styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
```

Comment:
Agreed. We need to start removing it since I think group consensus is we are no longer doing it and its not necessary to make the fluid bg color work.